pkg/build/controller/strategy/util.go

```diff
@@ -43,6 +43,11 @@ const (
 	// GitCloneContainer is the name of the container that will clone the
 	// build source repository and also handle binary input content.
 	GitCloneContainer = "git-clone"
+
+	// buildVolumeMountPath is where user defined BuildVolumes get mounted
+	buildVolumeMountPath = "/var/run/openshift.io/volumes"
```

Comment:
yeah these 2 constants could also be candidates for https://github.com/openshift/runtime-utils

Reply:
@adambkaplan said "nay" to adding the comments in that repo ... not sure I entirely agree but I'll defer ... I don't seem to have the resolve button for this one on comments I made @coreydaley but if you do feel free to mark this resolved